### PR TITLE
Update ReadME

### PR DESCRIPTION
**Pull request by mollieru**
Updates `-assets` ReadME language

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,16 @@
...
-For the initial alpha release, included are font files and an editable .ai and .eps file, as well as a .png for each of pattern on the site. In future releases, we will work to include .psd and Sketch files to support broader software usage. 
+For the alpha release, the following file types are available for your use:
+- .AI (Adobe Illustrator) 
+- .EPS files
+- Font files (Fontawesome, Merriweather, Source Sans Pro)
+- Color Swatches
+We encourage and solicit your input. Please leave any feedback in the project's primary repo here: https://github.com/18F/web-design-standards
 
-We encourage and solicit your feedback. Please leave any comments in the project's primary repo: https://github.com/18F/govt-wide-patternlibrary/issues
+### Fonts
```
Comment (mollieru) [inline README.md]: @maya @jpyuda should we add language about Font-Awesome's license here as well or is it sufficiently covered in the main repo?

Comment (maya) [inline README.md]: I think it's covered in the LICENSE already cc @brittag

Comment (maya) [inline README.md]: oh sorry, I thought this was the main repo. You probably should add it here.

Comment (mollieru) [inline README.md]: @maya ok copied font-awesome language from the main repo. everything else look ok here?

Comment (maya) [inline README.md]: @mollieru still working on the other repo, but will let you know!

**Comment by maya** [comment]
@mollieru updated the above in 5f40532 to match our repo and add correct license attribution.

**Pull request merged**